site/src/pages/HealthPage/WorkspaceProxyPage.tsx

```diff
@@ -0,0 +1,146 @@
+import { Alert } from "components/Alert/Alert";
+import { Helmet } from "react-helmet-async";
+import { pageTitle } from "utils/page";
+
+export const WorkspaceProxyPage = () => {
+  const healthStatus = useOutletContext<HealthcheckReport>();
+  const { workspace_proxy } = healthStatus;
+  const { regions } = workspace_proxy.workspace_proxies;
+  const theme = useTheme();
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("Workspace Proxy - Health")}</title>
+      </Helmet>
+
+      <Header>
+        <HeaderTitle>
+          <HealthyDot severity={workspace_proxy.severity} />
+          Workspace Proxy
+        </HeaderTitle>
+      </Header>
+
+      <Main>
+        {workspace_proxy.warnings.map((warning) => {
+          return (
+            <Alert key={warning.code} severity="warning">
+              {warning.message}
+            </Alert>
+          );
+        })}
+
+        {regions.map((region) => {
+          const warnings = region.status?.report?.warnings ?? [];
+
+          return (
+            <div
+              key={region.id}
+              css={{
+                borderRadius: 8,
+                border: `1px solid ${theme.palette.divider}`,
+                fontSize: 14,
+              }}
+            >
+              <header
+                css={{
+                  padding: 24,
+                  display: "flex",
+                  alignItems: "center",
+                  justifyContent: "space-between",
+                  gap: 24,
+                }}
+              >
+                <div css={{ display: "flex", alignItems: "center", gap: 24 }}>
+                  <div
+                    css={{
+                      width: 36,
+                      height: 36,
+                      display: "flex",
+                      alignItems: "center",
+                      justifyContent: "center",
+                    }}
+                  >
+                    <img
+                      src={region.icon_url}
+                      css={{ objectFit: "fill", width: "100%", height: "100%" }}
+                      alt=""
+                    />
+                  </div>
+                  <div css={{ lineHeight: "160%" }}>
+                    <h4 css={{ fontWeight: 500, margin: 0 }}>
+                      {region.display_name}
+                    </h4>
+                    <span css={{ color: theme.palette.text.secondary }}>
+                      {region.version}
+                    </span>
+                  </div>
+                </div>
+
+                <div css={{ display: "flex", flexWrap: "wrap", gap: 12 }}>
+                  {region.wildcard_hostname && (
+                    <Tooltip title="Wildcard Hostname">
+                      <Pill icon={<PublicOutlined />}>
+                        {region.wildcard_hostname}
+                      </Pill>
+                    </Tooltip>
+                  )}
+                  {region.version && (
```

Comment:
What is the type of `region.version`? Is it always a string?
Mainly just worried about random 0's showing up in the UI if this is potentially a number

Reply:
No, this should always be a string:

https://github.com/coder/coder/blob/main/coderd/database/dump.sql#L518-L519

Reply:
Yeap, it is always a string.

Reply:
Sweet. Just making sure the conditional logic didn't have any edge cases in rendering